BLAG.App/BLAG.App.UWP/BLAG.App.UWP.csproj

```diff
@@ -11,7 +11,7 @@
     <AssemblyName>BLAG.App.UWP</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.16299.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.17134.0</TargetPlatformVersion>
```

Comment:
Don't commit changes to files in projects you are not working on, this is xamarin